corehq/apps/cloudcare/static/cloudcare/js/formplayer/menus/controller.js

```diff
@@ -157,8 +129,8 @@ hqDefine("cloudcare/js/formplayer/menus/controller", function () {
         var queryResponse = menuResponse.queryResponse;
         if (sidebarEnabled && menuResponse.type === "entities" && queryResponse)  {
             let queryCollection = new Collection(queryResponse.displays);
-            if (queryResponse.hasOwnProperty("groupHeaders")) {
-                queryCollection = new Collection(groupDisplays(queryResponse.displays, queryResponse.groupHeaders));
+            if (Object.keys(queryResponse.groupHeaders).length > 0) {
```

Comment:
I'm not entirely sure, but do we still need to first check `queryResponse` contains `groupHeaders`? I think there would be a transient moment during deploy where FP changes aren't deployed yet so queryResponse doesn't contain `groupHeaders` and this would throw an error.

I think ideally, formplayer would be deployed first and then HQ the next week but it's most likely happening at the same time for this feature.

Reply:
Good. Point I will put that back in